common/map_file/nodes/map_param_loader/map_param_loader.cpp

```diff
@@ -0,0 +1,90 @@
+
+#include <ros/ros.h>
+#include <tf2_ros/static_transform_broadcaster.h>
+#include <tf2/LinearMath/Vector3.h>
+#include <tf2/LinearMath/Matrix3x3.h>
+#include <tf2/LinearMath/Transform.h>
+#include <tf2/LinearMath/Quaternion.h>
+#include <geometry_msgs/TransformStamped.h>
+
+#include <lanelet2_extension/projection/local_frame_projector.h>
+#include <lanelet2_extension/io/autoware_osm_parser.h>
+
+void printUsage()
+{
+  ROS_ERROR_STREAM("Usage:");
+  ROS_ERROR_STREAM("rosrun map_file map_param_loader [.OSM]");
+}
+
+// Get transform from base(ECEF) to target(map) using local_projector
+tf2::Transform getTransform(const std::string& base_frame, const std::string& target_frame)
```

Comment:
All logic functions need to be unit tested. This should be refactored into another file that can be unit tested. Then there should be some unit tests written to exercise it. This applies to all new functions in this PR which are ros independent.